ide/html.editor/src/org/netbeans/modules/html/editor/CssPreferences.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.netbeans.modules.html.editor;
+
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.prefs.PreferenceChangeEvent;
+import java.util.prefs.PreferenceChangeListener;
+import java.util.prefs.Preferences;
+import org.netbeans.api.editor.mimelookup.MimeLookup;
+import org.netbeans.modules.css.lib.api.CssTokenId;
+import org.openide.util.WeakListeners;
+
+/**
+ *
+ * @author mfukala@netbeans.org
+ */
+public class CssPreferences {
```

Comment:
where did this class come from? I suppose a copy from another module, just renamed?

Reply:
It is a reduced copy from `css.editor`. I neither want to add more to the exposed API nor make `html.editor` using an implementation dependency.

Reply:
understood